admin-manual/installation-setup/customization/antivirus-admin.rst

```diff
@@ -159,5 +159,5 @@ environment itself.
 .. _ClamAV manual pages: https://manpages.debian.org/jessie/clamav-daemon/clamd.conf.5.en.html
 .. _Compose .yml file: https://github.com/artefactual/archivematica/blob/qa/1.x/hack/docker-compose.yml
 .. _configuration file: https://github.com/artefactual-labs/ansible-clamav/blob/master/defaults/main.yml
-.. _Ansible environment variables: https://github.com/artefactual-labs/ansible-archivematica-src/tree/d4474c3dbaef2b561c87e0650c6ee386be6910a7#environment-variables
+.. _Ansible environment variables: https://github.com/artefactual-labs/ansible-archivematica-src/blob/stable/1.15.x/README.md
```

Comment:
```suggestion
.. _Ansible environment variables: https://github.com/artefactual-labs/ansible-archivematica-src/blob/stable/1.15.x/README.md#environment-variables
```

Reply:
Hhmm depending on how many links use fragments across the docs maybe it's worth figuring it out.

Reply:
Oh I think I got it :sparkles: 

`r"((.md)[#][\w-]*)"`

Reply:
Hmm - it's not quite it yet. :construction:

Reply:
ok now I think I got it, after some more testing - https://github.com/artefactual/archivematica-docs/pull/440/commits/4edd720320fa60be1d05ee0d1f2ac31379451fdf#diff-6e03d3d89630aba3403afea3514a3ac5948e70d1ced091012e56446f614da340R377

Reply:
:exploding_head: 

Great. Please don't forget to squash and merge.